dependency_support/dependency_support.bzl

```diff
@@ -37,6 +37,7 @@ load("//dependency_support/com_opencircuitdesign_netgen:com_opencircuitdesign_ne
 load("//dependency_support/edu_berkeley_abc:edu_berkeley_abc.bzl", "edu_berkeley_abc")
 load("//dependency_support/net_invisible_island_ncurses:net_invisible_island_ncurses.bzl", "net_invisible_island_ncurses")
 load("//dependency_support/net_sourceforge_ngspice:net_sourceforge_ngspice.bzl", "net_sourceforge_ngspice")
+load("//dependency_support/net_sourceforge_prdownloads_swig:net_sourceforge_prdownloads_swig.bzl", "net_sourceforge_prdownloads_swig")
```

Comment:
Why `net_sourceforge_prdownloads_swig` and not `org_swig`?

Reply:
Workspace names like this are a part of the public API of the package: Its name will be mentioned in all of the build files of all of the projects that use swig via rules_hdl. For this reason, changing a workspace name is difficult or impossible to do.

I think `org_swig` is a better choice here, because if SWIG would change where they host their files, it doesn't make sense that we need to break our public API for that reason. What matters to the user is that this package is SWIG, not where it was downloaded from. I have used net_sourceforce_* as a prefix only when there was no dedicate domain name to use.